test/module/irohad/simulator/simulator_test.cpp

```diff
@@ -261,35 +266,45 @@ TEST_F(SimulatorTest, FailWhenSameAsProposalHeight) {
 TEST_F(SimulatorTest, RightNumberOfFailedTxs) {
   // create a 3-height proposal, but validator returns only a 2-height verified
   // proposal
-  auto tx = shared_model::proto::TransactionBuilder()
-                .createdTime(iroha::time::now())
-                .creatorAccountId("admin@ru")
-                .addAssetQuantity("coin#coin", "1.0")
-                .quorum(1)
-                .build()
-                .signAndAddSignature(
-                    shared_model::crypto::DefaultCryptoAlgorithmType::
-                        generateKeypair())
-                .finish();
-
-  std::vector<shared_model::proto::Transaction> txs = {tx, tx, tx};
+  const int kNumTransactions = 3;
+  std::vector<shared_model::proto::Transaction> txs;
+  for (int i = 0; i < kNumTransactions; ++i) {
+    txs.emplace_back(shared_model::proto::TransactionBuilder()
+                         .createdTime(iroha::time::now() + i)
```

Comment:
I guess we have a special wrapper for this, do we?

Reply:
I think it is convenient to increment the value regardless of the units of time it uses. If you know a clearer way of doing this, please suggest it.

Reply:
I finally found this - we have a ```getUniqueTime()``` function for exactly this task. Unfortunately it has 2 implementation and both of them are out of this test scope. It should be refactored but not here I think.